### PR TITLE
Holy priest: Updated 'Mana efficiency' tab and added support for new talent

### PR DESCRIPTION
**Pull request by emilnormann**
Fixed bugs and added new Dragonflight functionality to 'Mana efficiency' tab and added support for Revitalizing Prayers

### CODE REVIEW

File: src/analysis/retail/priest/holy/modules/features/HolyPriestHealingEfficiencyTracker.tsx [hunk 1]
```diff
@@ -86,28 +132,31 @@ class HolyPriestHealingEfficiencyTracker extends HealingEfficiencyTracker {
...
+              healingSpellId
+            ].effectiveHealing;
+            spellInfo.overhealingDone += this.echoOfLight.masteryHealingBySpell[
+              healingSpellId
+            ].overHealing;
+          }
+        });
+      }
+    } catch {
+      return spellInfo; //Avoids crashes
```
Comment (ToppleTheNun) [inline src/analysis/retail/priest/holy/modules/features/HolyPriestHealingEfficiencyTracker.tsx:165]: What crashes were occurring?

Comment (emilnormann) [reply src/analysis/retail/priest/holy/modules/features/HolyPriestHealingEfficiencyTracker.tsx:165]: I am not sure the exact reason but it passed empty spellinfo with some different spells like dispel magic and divine star where they weren't used. No healing is lost so I figured I might as well fix it like this.

File: src/analysis/retail/priest/holy/modules/spells/Renew.tsx [hunk 2]
```diff
@@ -132,6 +151,13 @@ class Renew extends Analyzer {
   }
 
   handleRenewApplication(event: ApplyBuffEvent | RefreshBuffEvent) {
+    this.timestampOfLastRenewApplication = event.timestamp;
+    if (this.timestampOfLastRenewApplication - this.timestampOfLastPrayerCast < MS_BUFFER) {
+      console.log(this.timestampOfLastRenewApplication - this.timestampOfLastPrayerCast);
```
Comment (ToppleTheNun) [inline src/analysis/retail/priest/holy/modules/spells/Renew.tsx]: Can you remove this `console.log`?

**Review by ToppleTheNun — CHANGES_REQUESTED** [review]
One small change then should be good to go.

**Review by ToppleTheNun — APPROVED** [review]

**Comment by danny-janse** [comment]
I see quite a few of unused methods that have been added. Do you have any plans to them?

**Review by ToppleTheNun — APPROVED** [review]

**Pull request merged**